steer_drive_controller/CMakeLists.txt

```diff
@@ -0,0 +1,103 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(steer_drive_controller)
+
+find_package(catkin REQUIRED COMPONENTS
+  controller_interface
```

Comment:
please sort these dependencies

(in vim you can do it by selecting these lines in visual mode and `:sort` )

Reply:
agreed